app/views/LicensorViewComponent.vue

```diff
@@ -346,6 +412,38 @@ module.exports = Vue.extend({
       catch err
         console.log(err)
         forms.setErrorToProperty(el, 'updateClientFeatures', 'Something went wrong')
+    
+    onEditApiClient: co.wrap ->
+      el = $('#client-form')
+      requiredProps = ['clientName', 'minimumLicenseDays', 'manageLicensesViaUI', 'manageLicensesViaAPI', 'revokeLicensesViaUI', 'revokeLicensesViaAPI', 'manageSubscriptionViaAPI', 'revokeSubscriptionViaAPI']
+      data = @runValidation(el, requiredProps)
+      unless data
+        return
+
+      try
+        apiClient = yield api.apiClients.getByName(data.clientName)
+        unless apiClient.length>0
+          forms.setErrorToProperty(el, 'editClient', 'API client not found')
+          return
+        attrs = {
+          name: data.clientName
+          minimumLicenseDays: parseInt(data.minimumLicenseDays)
+          permissions: {
+            manageLicensesViaUI: (data.manageLicensesViaUI == 'true')
+            manageLicensesViaAPI: (data.manageLicensesViaAPI == 'true')
+            revokeLicensesViaUI: (data.revokeLicensesViaUI == 'true')
+            revokeLicensesViaAPI: (data.revokeLicensesViaAPI == 'true')
+            manageSubscriptionViaAPI: (data.manageSubscriptionViaAPI == 'true')
+            revokeSubscriptionViaAPI: (data.revokeSubscriptionViaAPI == 'true')
+          }
+        }
+        attrs.id = apiClient[0]._id
+        apiClient = yield api.apiClients.editClient(attrs)
+        alert("API client updated")
```

Comment:
Can use noty here instead for better UX:
        `noty text: 'API Client updated', timeout: 3000, type: 'success'`